connector/saml/saml.go

```diff
@@ -106,6 +104,9 @@ type Config struct {
 	//		urn:oasis:names:tc:SAML:2.0:nameid-format:persistent
 	//
 	NameIDPolicyFormat string `json:"nameIDPolicyFormat"`
+
+	// Allow a clock_drift when checking dates
+	AllowedClockDrift int64 `json:"allowedClockDrift"`
```

Comment:
5 minutes seems like a lot, we do 30 seconds in our Go client[0]. Can we start with 30 second value, ensure we print good errors when the times disagree, and then let users open bugs?

[0] https://github.com/coreos/go-oidc/blob/2cc7913f9f6f26eff95b633c29bfe04895d38121/jwks.go#L17-L25